test/runtests.jl

```diff
@@ -5,8 +5,10 @@ using Compat
 using BinDeps
 
 Pkg.add("Cairo")  # Tests apt-get code paths
+Pkg.build("Cairo")
```

Comment:
Not when it's already installed, i.e., if you're running the BinDeps tests locally.